src/components/ProductPage/ProductIndexComponentsTable.tsx

```diff
@@ -37,7 +39,13 @@ const ProductIndexComponentsTable: React.FC<ProductIndexComponentsProps> = ({
     )
   }
 
-  if (components === undefined || components.length === 0) {
+  if (chainId && chainId === 137) {
```

Comment:
maybe setup a constants file with chain IDs so this is `chainID === CHAIN_IDS.POLYGON`